plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -13,51 +18,247 @@
+        data: function() {
+            return {
+                jsonerror: "",
+                jsonstr: ""
+            };
+        },
+        computed: {
+            centerDialogVisible: function() {
+                return this.$store.getters["countlyRemoteConfig/parameters/showJsonEditor"];
+            }
+        },
+        methods: {
+            validateJson: function() {
+                this.jsonerror = "";
+                try {
+                    // try to parse
+                    if (this.jsonstr) {
+                        JSON.parse(this.jsonstr);
+                    }
+                }
+                catch (e) {
+                    this.jsonerror = JSON.stringify(e.message);
+                }
+            },
+            prettyFormat: function() {
+                // reset error
+                var jsonValue = "";
+                try {
+                    // try to parse
+                    jsonValue = JSON.parse(this.jsonstr);
+                    this.jsonstr = JSON.stringify(jsonValue, null, 2);
+                }
+                catch (e) {
+                    // Do nothing
+                }
+            },
+            submit: function() {
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditor", false);
+            },
+            cancel: function() {
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditor", false);
+            }
+        }
+    });
     var ConditionsDrawer = countlyVue.views.create({
         template: CV.T("/remote-config/templates/conditions-drawer.html"),
+        components: {
+            "color-tag": ColorTag
+        },
         props: {
             controls: {
                 type: Object
             }
         },
+        computed: {
+            showSeedValue: function() {
+                if (!_.isEmpty(this.managedPropertySegmentation.queryText) && this.managedPropertySegmentation.queryText.includes("Random Percentile")) {
+                    return true;
+                }
+                return false;
+            }
+        },
         data: function() {
+            var additionalProperties = [];
+            var remoteConfigFilterRules = [new countlyQueryBuilder.RowRule({
+                name: "cly.remote-config-random-percentile-rule",
+                selector: function(row) {
+                    return row.property.id === "condition.randomPercentile";
+                },
+                actions: [new countlyQueryBuilder.RowAction({
+                    id: "disallowOperator",
+                    params: {
+                        selector: function(operator) {
+                            return ["cly.=", "cly.!=", "cly.contains", "cly.between", "cly.isset"].includes(operator.id);
+                        }
+                    }
+                })]
+            })];
+            additionalProperties.push(new countlyQueryBuilder.Property({
+                id: 'condition.randomPercentile',
```

Comment:
please use the same id for random percentile as its on master - `up.random_percentile` i think to avoid compatibility issues.

Reply:
Yeah , sure.